src/panel.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { Fragment, useState } from 'react';
+import { useParameter } from '@storybook/api';
+import {
+  Form,
+  Placeholder,
+  SyntaxHighlighter,
+  TabsState,
+} from '@storybook/components';
+import { PARAM_KEY } from './constants';
+import { MockedResponse, Parameters } from './types';
+import type { OperationDefinitionNode } from 'graphql';
+
+const getOperationName = (mockedResponse: MockedResponse): string => {
+  if (mockedResponse.request.operationName) {
+    return mockedResponse.request.operationName;
+  }
+
+  const operationDefinition = mockedResponse.request.query.definitions.find(
+    (definition): definition is OperationDefinitionNode =>
+      definition.kind === 'OperationDefinition'
+  );
+
+  if (operationDefinition && operationDefinition.name) {
+    return operationDefinition.name.value;
+  }
+
+  return 'Unnamed';
+};
+
+export const ApolloClientPanel: React.FC = () => {
+  const [mockedResponse, setMockedResponse] = useState<MockedResponse | null>(
+    null
+  );
+  const { mocks = [] } = useParameter<Parameters>(PARAM_KEY, {});
+
```

Comment:
Optional UX improvement / future enhancement: if there is only one mock in the array, would it make sense to auto-select it? This caught me a few times when flipping back and forth between similar stories